editor/.dependency-cruiser.js

```diff
@@ -0,0 +1,371 @@
+module.exports = {
+  forbidden: [
+    /* rules from the 'recommended' preset: */
+    // {
+    //   name: 'no-circular',
```

Comment:
it would be _so good_ if we could enable this! however, right now it produces 253 errors :)